src/worker/storage.ts

```diff
@@ -59,7 +59,7 @@ export class PredictiveTextStudioDexie extends Dexie {
        * +------------------+
        */
       projectData:
-        "++id, langName, bcp47Tag, authorName, modelID, copyright, dictionaryName, version",
+        "++id, language, bcp47Tag, authorName, modelID, copyright, dictionaryName, version",
```

Comment:
This schema changes **requires** a database migration. Please see: https://dexie.org/docs/Tutorial/Design#database-versioning

## What does a migration look like?

You will leave this line as is. It represents the **original** database schema.

```suggestion
        "++id, langName, bcp47Tag, authorName, modelID, copyright, dictionaryName, version",
```

After `this.version(5)` you will have some code something like this:

```typescript
    this.version(6)
      .stores({  projectData:
        "++id, language, bcp47Tag, authorName, modelID, copyright, dictionaryName, version", })
      .upgrade((transaction) => {
        return transaction.table("projectData").toCollection().modify((project) => {
          project.language = project.langName;
          delete project.langName;
        });
      });
```

---

Although the stakes are fairly low for this project, when modifying an app with an existing database, it's best to do things in database migrations to _upgrade_ the database, rather than to change the schema and require the database to be recreated. It's especially crucial here, since the database exists in each users' browsers, and is not locally managed.

Reply:
Thanks for the insightful comment, learned a thing or two from that! 

That's a really nice feature from dexie, I never considered that front-ends could have databases that needed to be migrated :brain:

I have pushed the requested changes :+1: